lkmpg.tex

```diff
@@ -1293,6 +1327,12 @@ \section{sysfs: Interacting with your module}
 sudo rmmod hello_sysfs
 \end{codebash}
 
+In the above case, we use a simple kobject to create a directory under sysfs, and communicate with its attributes.
+The \cpp|kobject| structure first made its appearance in the 2.5.45 development kernel.
```

Comment:
You don't have to mention "2.5.45 development kernel." Instead, let's say, "since Linux v2.6.0."